.drone.yml

```diff
@@ -10,8 +10,10 @@ pipeline:
       - mkdir -p /go/bin
       - "curl -s https://glide.sh/get | sh"
       - glide install
+      - go get github.com/vektra/mockery/.../
```

Comment:
What version will this install? Latest? (if so that could be error prone – probably best to lock down the version)